eldritch/test_encounters.py

```diff
@@ -520,6 +624,343 @@ def testSociety4Fail(self):
+
+  def testChurch2(self):
+    self.state.event_stack.append(encounters.Church2(self.char))
+    self.resolve_until_done()
+    self.assertEqual(self.char.bless_curse, 1)
+
+  def testChurch2Curse(self):
+    self.char.bless_curse = -1
+    self.state.event_stack.append(encounters.Church2(self.char))
+    self.resolve_until_done()
+    self.assertEqual(self.char.bless_curse, 0)
+
+  def testChurch3money(self):
+    self.char.dollars = 7
+    self.state.event_stack.append(encounters.Church3(self.char))
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "Money")
+    self.resolve_until_done()
+    self.assertEqual(self.char.dollars, 3)
+
+  def testChurch4No(self):
+    self.state.event_stack.append(encounters.Church4(self.char))
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "No")
+    self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 0)
+    self.assertEqual(len(self.state.unique), 2)
+
+  def testChurch4Yes(self):
+    self.state.event_stack.append(encounters.Church4(self.char))
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "Yes")
+    self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 1)
+    self.assertEqual(self.char.possessions[0].name, "Holy Water")
+    self.assertEqual(len(self.state.unique), 1)
+
+  def testChurch5Fail(self):
+    self.state.event_stack.append(encounters.Church5(self.char))
+    self.char.sanity = 5
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=2)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 2)
+    self.assertEqual(self.char.place.name, "Church")
+
+  def testChurch5Pass1(self):
+    self.state.event_stack.append(encounters.Church5(self.char))
+    self.char.lore_luck_slider = 3
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 3)
+    self.assertEqual(self.char.place.name, "Southside")
+  
+  def testChurch5Pass2(self):
+    self.state.event_stack.append(encounters.Church5(self.char))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 5)
+    self.assertEqual(self.char.place.name, "Church")
+  
+  def testChurch6NoClues(self):
+    self.state.event_stack.append(encounters.Church6(self.char))
+    self.assertEqual(self.char.clues, 0)
+    self.resolve_until_done()
+    self.assertEqual(self.char.clues, 0)
+
+  def testChurch6CluesNo(self):
+    self.state.event_stack.append(encounters.Church6(self.char))
+    self.char.clues = 2
+    self.assertEqual(self.char.clues, 2)
+    choice = self.resolve_to_choice(MultipleChoice)
+    choice.resolve(self.state, "No")
+    self.resolve_until_done()
+    self.assertEqual(self.char.clues, 2)
+
+  def testChurch6CluesNo(self):
```

Comment:
You seem to have accidentally named this the same as the previous test.

Reply:
copy paste is still my friend